Automattic/vip-go-mu-plugins
# Updating gutenberg-ramp plugin to latest

## Description
Update to the latest version of `gutenberg-ramp`. 

The changes add a notice to `wp-admin` to let users know that the plugin is obsolete and is being phased out, along with a link to documentation. Also invoked is a deprecation function.

## Changelog Description

Changes are noted above.

See changes [here](https://github.com/Automattic/gutenberg-ramp/compare/a9987787502140e0c1c18118032dd9b4d6f87942...7d0cd89e5df5efec902cadc56ca0c844d7be4b7a). 

And Pull-Request [here](https://github.com/Automattic/gutenberg-ramp/pull/83).

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback) [ tested with standalone WP 5.6 ]
- [x] This change works and has been tested on a Go sandbox [ tested with vip-test site ]

## Steps to Test

1. Check out PR on a WordPress site
2. Enable `gutenberg-ramp` plugin using `gutenberg_ramp_load_gutenberg( true );`
2. Go to `wp-admin`
3. Notice is displayed.
